follow/src/main/resources/application.yml

```
@@ -50,14 +63,20 @@ management:
     tracing:
       endpoint: "http://localhost:9411/api/v2/spans"
   tracing:
+    enabled: true
     sampling:
       probability: 1.0
   endpoints:
     web:
       exposure:
-        include: ["*"]
+        include: [ "*" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)